### PR TITLE
LTD-1971: UI test for LU consolidate

### PR DESCRIPTION
**Pull request by wkeeling**

### CODE REVIEW

File: ui_tests/caseworker/features/give_advice.feature [hunk 1]
```diff
@@ -159,3 +159,32 @@ Feature: I want to record my user advice and any comments and conditions relatin
...
+    And I click submit recommendation
+    Then I see "reason for approving" as the reasons for approving
+    And I see "licence condition" as the licence condition
+    When I click "Finalise case"
+    And I click save
+    And I click "Generate"
+    And I select the template "SIEL template"
+    And I click continue
+    And I click preview
+    Then I see that "16. Control list no" is "ML1a" on the SIEL licence preview
```
Comment (saruniitr) [inline ui_tests/caseworker/features/give_advice.feature:186]: should we check for licence reference in the preview?

File: ui_tests/caseworker/features/give_advice.feature [hunk 2]
```diff
@@ -159,3 +159,32 @@ Feature: I want to record my user advice and any comments and conditions relatin
...
+    And I go to my case list
+    And I click the application previously created
+    And I click the recommendations and decision tab
+    And I click "Review and combine"
+    And I enter "reason for approving" as the reasons for approving
+    And I enter "licence condition" as the licence condition
+    And I click submit recommendation
+    Then I see "reason for approving" as the reasons for approving
+    And I see "licence condition" as the licence condition
+    When I click "Finalise case"
```
[the remark this answers is not in the document]
Comment (wkeeling) [reply ui_tests/caseworker/features/give_advice.feature:180]: Ashley has confirmed there will be a separate test for LU countersign.

File: ui_tests/caseworker/features/give_advice.feature (hunk 1 above)
Comment (wkeeling) [reply ui_tests/caseworker/features/give_advice.feature]: Check now added

File: ui_tests/caseworker/conftest.py [hunk 3]
```diff
@@ -360,6 +401,23 @@ def selected_created_template(driver, context):  # noqa
...
+@then("I see the licence number on the SIEL licence preview")
+def should_see_licence_number_on_siel_licence_preview(driver, context):  # noqa
+    text = GeneratedDocument(driver).get_document_preview_text()
+    assert f"{context.reference_code}-01" in text
+
+
+@then(parsers.parse('I see that "{item_name}" is "{value}" on the SIEL licence preview'))  # noqa
+def should_see_item_on_siel_licence_preview(driver, item_name, value):  # noqa
+    val = GeneratedDocument(driver).get_item_from_siel_document_preview(item_name)
+    assert val == value
```
Comment (saruniitr) [inline ui_tests/caseworker/conftest.py:418]: might as well remove `val`?

**Review by saruniitr — APPROVED** [review]

File: ui_tests/caseworker/conftest.py (hunk 3 above)
Comment (wkeeling) [reply ui_tests/caseworker/conftest.py]: Yep, done

**Pull request merged**